cita-jsonrpc/src/responser.rs

```diff
@@ -0,0 +1,83 @@
+// CITA
+// Copyright 2016-2018 Cryptape Technologies LLC.
+
+// This program is free software: you can redistribute it
+// and/or modify it under the terms of the GNU General Public
+// License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any
+// later version.
+
+// This program is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied
+// warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE. See the GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use config::Config;
+use get_build_info_str;
+use jsonrpc_types::rpctypes::SoftwareVersion;
+use jsonrpc_types::ErrorCode;
+use libproto::protos::response::Response;
+use libproto::router::{MsgType, RoutingKey, SubModules};
+use libproto::Message;
+use libproto::TryInto;
+use pubsub::channel::Sender;
+use serde_json;
+
+pub struct Responser {
+    tx: Sender<(String, Vec<u8>)>,
+    config: Config,
+}
+
+impl Responser {
+    pub fn new(config: Config, tx: Sender<(String, Vec<u8>)>) -> Responser {
+        Responser { tx, config }
+    }
+
+    pub fn reply_request(&self, _: RoutingKey, mut msg: Message) {
+        let req_opt = msg.take_request();
```

Comment:
`reply_request(&self, _: RoutingKey, mut msg: Message)`
According to my view, If the second parameter is useless, we could del it. if we need it actually,  a variable name should be given.

Reply:
Yes, it's useless in this case. Considering scalability, I think we should keep it.